documents/charter.md

```diff
@@ -1,23 +1,24 @@
-Video traffic is already 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
+Video traffic is 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
 Across developed and emerging markets video traffic forms 50-80% of traffic volume on mobile networks.
 New formats like short form videos have seen tremendous growth in recent years.
 These growth trends are likely to increase with new populations coming online on mobile-first markets.
 
-Mobile network operators continuously invest in network resources, including deployment of new generations or new bands of spectrum.
-Since spectrum is a limited and expensive resource, operators often make use of flow-based traffic handling such as shaping of video traffic, especially when the network is highly loaded.
-Operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
+While mobile network operators continuously invest in new network capacity, including deployment of new generations or new bands of spectrum, these efforts are not keeping pace with growth in demand. These network operators have found it faster and less expensive to invest in shaping of video traffic on a per-flow basis, which negatively affects video stream quality. Because that traffic is encrypted, network operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
```

Comment:
/ be so volatile as to rapidly change the maximum throughput throughout the course of a session/ 
- true I expect of some technologies; however, is it wiser to focus towards what capacity a session can expect, e.g., to say something like:
/ radio propagation condition may be volatile, changing the maximum throughput available to a session/

Reply:
Hi, @gorryfair, let's start by making sure I understand your suggestion - given the previous proposed text

> Local mobile radio conditions may constrain the maximum throughput for a given client, or be so volatile as to rapidly change the maximum throughput throughout the course of a session.

and this proposed text

> radio propagation condition may be volatile, changing the maximum throughput available to a session

Is the resulting text something like this? 

> Local mobile radio conditions may constrain the maximum throughput for a given client, or or may be volatile, changing the maximum throughput available to a session.

If I got that right - I'm reading the previous proposed text to describe frequent updates to SCONEPRO guidance, and the new proposed text to describe less frequent updates to SCONEPRO guidance (with the minimum being that an operator provides conservative SCONEPRO guidance once, so that the session need not adapt because of fluctuations in radio conditions). 

Is that the intent?

Reply:
I thought something like that might be a useful option to allow. I realise there are radio/mobile people who may have opinions also.

Reply:
@gorryfair, one thing that I'm not sure is captured in the proposed charter, but keeps coming up in discussions, is that the network properties provided to the application are often based on actual path capacity (especially for paths that traverse wireless links), but can also be based on subscriptions. I'd expect network properties to change more often if they are based on path capacities than if they are based on subscriptions. I'll see what I can do, to make this clearer in the charter.